presto-yarn-package/src/main/resources/appConfig-test-no-catalog.json

```diff
@@ -17,7 +17,7 @@
     "site.global.jvm_args": "['-server', '-Xmx1024M', '-XX:+UseG1GC', '-XX:G1HeapRegionSize=32M', '-XX:+UseGCOverheadLimit', '-XX:+ExplicitGCInvokesConcurrent', '-XX:+HeapDumpOnOutOfMemoryError', '-XX:OnOutOfMemoryError=kill -9 %p']",
 
     "application.def": ".slider/package/PRESTO/${app.package.name}.zip",
-    "java_home": "/usr/lib/jvm/java"
```

Comment:
why did it have to change? It seems unreleated to this change. I would prefer to have new installed in the same as old one was installed.